inc/saidebugcounter.h

```diff
@@ -81,6 +81,9 @@ typedef enum _sai_debug_counter_bind_method_t
  */
 typedef enum _sai_in_drop_reason_t
 {
+    /** Start of in drop reasons */
+    SAI_IN_DROP_REASON_START,
```

Comment:
sai_in_drop_reason_t: first enum should start with zero
assign this value to the first item, check _sai_hostif_trap_type_t